scripts/download_model.sh

```diff
@@ -5,7 +5,7 @@ mkdir -p ${MODEL_DIR}
 if [ ! -f "${MODEL_DIR}/AudioCLIP-Full-Training.pt" ]; then
   echo "Downloading model"
   echo "------ Download AudioCLIP model ------"
-  wget -q https://github.com/AndreyGuzhov/AudioCLIP/releases/download/v0.1/AudioCLIP-Full-Training.pt
+  wget https://github.com/AndreyGuzhov/AudioCLIP/releases/download/v0.1/AudioCLIP-Full-Training.pt
```

Comment:
what's the motivation for this?

Reply:
As the model is relatively large, user might doubt whether the downloading is stuck if they cannot see the process.(I downloaded the model for more than 5 minutes at home as the network is not so stable, so I think other users might meet up with similar problem if they cannot see the process)